### PR TITLE
Site Settings SEO: Improved Error Message Design

### LINKED ISSUE #5498: Plans SEO: Improve Design of Paste Error Message
Care of @drw158:

Found this error that doesn't fit any of our error styles:

screen shot 2016-05-20 at 11 03 13 am

I think it should be either the inline error, or the global error.

Generally I think I like the inline errors over the global errors, so perhaps we could use those for the other errors too.

I think I'd rather just merge this as is and fix it later since it's a design issue, and I'd like to get the Call for Testing out soon.

Inline error 

screen shot 2016-05-20 at 12 29 32 pm

### PR DESCRIPTION
**Pull request by roundhill**
Use `FormInputValidation` component for copy/paste error message.

**To test:** Try and type in one of the site verification fields, you should see this error:

screen shot 2016-05-24 at 10 22 20 am

@drw158 needs review! I think you mentioned doing the error inline, but it doesn't look like the text input with affixes component supports errors at this time.

Fixes #5498

### CODE REVIEW

**Comment by rickybanister** [comment]
It'd be nice if the field in error could have some indication of the error. A red focus ring would be nice, but that doesn't help color blind folks at all. Having the error message inline would definitely help.

**Comment by drw158** [comment]
LGTM I wonder if we can update the text input component to support inline errors.

**Comment by roundhill** [comment]
> LGTM I wonder if we can update the text input component to support inline errors.

I actually figured out a pretty good way to support it:
screen shot 2016-05-24 at 3 18 11 pm

It will also show the invalid verification code error when you attempt to save the form, too:
screen shot 2016-05-24 at 3 20 50 pm

Give it a try!

**Comment by drw158** [comment]
I like it! A lot better.

File: client/my-sites/site-settings/form-seo.jsx [hunk 1]
```diff
@@ -367,18 +374,23 @@ export const SeoForm = React.createClass( {
...
 				</Card>
 			</div>
 		);
+	},
+
+	getVerificationError( isPasteError ) {
+		return (
+			<FormInputValidation isError={ true } text={
+				isPasteError
+					? this.translate( 'Verification code should be copied and pasted in.' )
```
Comment (rralian) [inline client/my-sites/site-settings/form-seo.jsx]: copy suggestion
> Verification code should be copied and pasted into this field.

File: client/my-sites/site-settings/form-seo.jsx [hunk 2]
```diff
@@ -204,6 +207,7 @@ export const SeoForm = React.createClass( {
 
 		const isSitePrivate = parseInt( blog_public, 10 ) !== 1;
 		const isDisabled = isSitePrivate || isSubmittingForm;
+		const isSaveDisabled = isDisabled || isSubmittingForm || ( ! showPasteError && invalidCodes.length > 0 );
```
Comment (rralian) [inline client/my-sites/site-settings/form-seo.jsx]: I mentioned this in another issue https://github.com/Automattic/wp-calypso/issues/5612 but the entire form should be disabled while the settings are loading. This is generally done with a `disabled` attribute on every field.

**Comment by rralian** [comment]
I'm not sure about showing an error when a user types into the field, but it's worth waiting for user feedback rather than debating here.

One thing though, it would be nice if the error message was removed when an input loses focus. Not a blocker here though.

I had one copy suggestion, but otherwise this is good to 🚢

**Pull request merged**